pkg/ssh/auth/agent.go

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2017 The Kubicorn Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auth
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+
+	"golang.org/x/crypto/ssh"
+	"golang.org/x/crypto/ssh/agent"
+)
+
+// SystemAgent returns system agent if it exists.
+func SystemAgent() agent.Agent {
```

Comment:
So wondering how the workflow would work here..

Should we return an error if the system agent doesnt exist? 
Should we change the function name to `SystemAgentIfExists()` or something?

Just think we could be a little more strict here is all

Reply:
> Should we return an error if the system agent doesnt exist?

It could be useful for debugging purposes, so I added it.

> Should we change the function name to SystemAgentIfExists() or something?

Sounds better than `SystemAgent`. Updated.